app/src/main/java/org/thoughtcrime/securesms/ui/Components.kt

```diff
@@ -89,13 +110,69 @@ fun <T> OptionsCard(card: OptionsCard<T>, callbacks: Callbacks<T>) {
 }
 
 
+@Composable
+fun ItemButton(
+    @StringRes textId: Int,
+    @DrawableRes icon: Int,
+    colors: ButtonColors = transparentButtonColors(),
+    @StringRes contentDescription: Int = textId,
+    onClick: () -> Unit
+) {
+    ItemButton(stringResource(textId), icon, colors, stringResource(contentDescription), onClick)
+}
+
+@Composable
+fun ItemButtonWithDrawable(
+    @StringRes textId: Int,
+    @DrawableRes icon: Int,
+    colors: ButtonColors = transparentButtonColors(),
+    @StringRes contentDescription: Int = textId,
+    onClick: () -> Unit
+) {
+    val context = LocalContext.current
+
+    ItemButton(
+        text = stringResource(textId),
+        icon = {
+            Image(
+                painter = rememberDrawablePainter(drawable = AppCompatResources.getDrawable(context, icon)),
+                contentDescription = stringResource(contentDescription),
+                modifier = Modifier.align(Alignment.Center)
+            )
+        },
+        colors = colors,
+        onClick = onClick
+    )
+}
+
 @Composable
 fun ItemButton(
     text: String,
     @DrawableRes icon: Int,
     colors: ButtonColors = transparentButtonColors(),
     contentDescription: String = text,
     onClick: () -> Unit
+) {
+    ItemButton(
+        text = text,
+        icon = {
+            Icon(
+                painter = painterResource(id = icon),
+                contentDescription = contentDescription,
```

Comment:
Same here in regards to the contentDescription. How would a screen reader interpret this?

Reply:
gone